app/models/inventory_type.rb

```diff
@@ -2,4 +2,15 @@ class InventoryType < ApplicationRecord
   has_many :box_items
   has_many :core_box_items
   has_many :inventory_tallies
+
+  after_create :create_inventory_tallies
+
+  private
+
+  def create_inventory_tallies
+    locations = Location.where(location_type: "storage_unit")
+    locations.each do |location|
+      inventory_tallies.create(storage_location: location)
```

Comment:
Thanks for working on this, @juliacfurlani !!!

We'll need to do one more loop here so each Location gets and InventoryTally for each InventoryType.

(If this codespeak is helpful: under `locations.each do`, we need `InventoryType.all.each do` and then the `inventory_tallies.create` can include the second param of `inventory_type: inventory_type`)

Would you be willing to make that change and update the tests?

Reply:
Sure, no problem!!